architecture/adr-006-skeleton-screens.md

```diff
@@ -0,0 +1,36 @@
+# Architecture Decision Record: Skeleton Screens
```

Comment:
Documentation is always a 👍

Reply:
Does (some of) this belong in the design system? Parts of this refer to a package - `skeleton-screen-css` which is not even a direct dependency of this package.

It seems like a decision which really spans the two projects. Using skeleton screens is something related to the async/clientside nature of the React components but the effect of using use `skeleton-screen-css` goes in the design system. We even have stories for skeleton elements in the design system. 🤔

Reply:
Well another problem connected to the fragmented architecture.
I'll move it to the design system and make a PR there.
Easy peasy no problem.